package.json

```diff
@@ -9,30 +9,67 @@
     "url": "https://github.com/zendeskgarden/tailwindcss/issues"
   },
   "version": "0.0.0",
-  "main": "src/index.js",
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "build": "true",
+    "build": "rollup -c rollup.config.js",
     "format": "prettier-package-json --write && yarn format:all --write",
     "format:all": "prettier --loglevel warn '**/*.{js,ts,json,md}' '!CHANGELOG.md' '!dist/**'",
     "lint": "yarn lint:js && yarn lint:md",
     "lint:js": "eslint src/ --ext js,ts --max-warnings 0",
     "lint:md": "markdownlint README.md src/**/*.md",
     "prepare": "yarn build",
     "tag": "[ `git rev-parse --abbrev-ref HEAD` = 'master' ] && standard-version --no-verify",
-    "test": "prettier-package-json --list-different && yarn format:all --check && yarn lint"
+    "test": "jest",
+    "test:ci": "prettier-package-json --list-different && yarn format:all --check && yarn lint && yarn test"
+  },
+  "types": "dist/typings/index.d.ts",
+  "dependencies": {
+    "@zendeskgarden/css-bedrock": "^8.0.1",
+    "@zendeskgarden/react-theming": "^8.14.0",
```

Comment:
Should we open up a wider range here? I think anything over 8.0 would work.

Reply:
Or maybe `styled-components` is a limiting factor?

Reply:
Since this isn't a peer dependency we can have the highest value possible here. The open range will allow just allow bug fixes to be pulled in without a valid range.

With `v8.14` having the new color palettes I think it could be a sane bottom value.